### PR TITLE
Update Group Policy Central Store part

### PR DESCRIPTION
**Pull request by MK-Kamal**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3772

### CODE REVIEW

**Comment by e0i** [comment]
@mapalko The PR that addresses issue #3772 is available for review. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ ms.reviewer:
 -   Certificate trust
 
 
-You need a Windows 10, version 1703 workstation to run the Group Policy Management Console, which provides the latest Windows Hello for Business and PIN Complexity Group Policy settings.  To run the Group Policy Management Console, you need to install the Remote Server Administration Tools for Windows 10. You can download these tools from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?id=45520).
-Install the Remote Server Administration Tools for Windows 10 on a computer running Windows 10, version 1703.
+To get the latest Windows Hello for Business and PIN Complexity Group Policy settings , you need to create a Group Policy Central Store and update your Group Policy Central Store ADMX files with the latest Administrative Templates files . See [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administrative-templates-in-windows) .
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md]:
```suggestion
To get the latest Windows Hello for Business and PIN Complexity Group Policy settings, you need to create a Group Policy Central Store and update your Group Policy Central Store ADMX files with the latest Administrative Template files. See [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administrative-templates-in-windows).
```

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ ms.reviewer:
...
+
+Alternatively,You need a Windows 10, version 1703 workstation to run the Group Policy Management Console, To run the Group Policy Management Console, you need to install the Remote Server Administration Tools for Windows 10. You can download these tools from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?id=45520). Install the Remote Server Administration Tools for Windows 10 on a computer running Windows 10, version 1703. Or Alternatively you can create copy the .ADMX and .ADML files from a Windows 10, version 1703 to their respective language folder on a Windows Server
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md]:
```suggestion
Alternatively, you need a Windows 10, version 1703 workstation to run the Group Policy Management Console.  To run the Group Policy Management Console, you need to install the Remote Server Administration Tools for Windows 10. You can download these tools from the [Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=45520). Install the Remote Server Administration Tools for Windows 10 on a computer running Windows 10, version 1703. Or you can copy the .ADMX and .ADML files from a computer running Windows 10, version 1703 to their respective language folder on a Windows Server.
```

**Comment by MK-Kamal** [comment]
@nenonix Confirmed Thank you!

**Comment by e0i** [comment]
@mapalko Following up for updates.

**Comment by e0i** [comment]
@mapalko Following up for updates.

**Comment by e0i** [comment]
The PR is under escalation.

**Comment by illfated** [comment]
This file was amended in commit 5bde60c9d7091f610fd0c95f036e7a1cffcb3cb2 . (https://github.com/MicrosoftDocs/windows-itpro-docs/commit/5bde60c9d7091f610fd0c95f036e7a1cffcb3cb2#diff-be35902ff7f05711df8385f9e3db7920)
Change:
```diff
- You need a Windows 10, version 1703 workstation to run the Group Policy Management Console, which provides the latest Windows Hello for Business and PIN Complexity Group Policy settings.  To run the Group Policy Management Console, you need to install the Remote Server Administration Tools for Windows 10. You can download these tools from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?id=45520).
+ You need a Windows 10, version 1703 workstation to run the Group Policy Management Console, which provides the latest Windows Hello for Business and PIN Complexity Group Policy settings.  To run the Group Policy Management Console, you need to install the Remote Server Administration Tools for Windows 10. You can download these tools from the [Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=45520).
```
(removal of `/en-us/` from the URL)

**Comment by illfated** [comment]
To be precise, this only affects the originating file, not the result of this PR change. Even so, it still causes a file conflict in the file history, before a rebase of this PR content (`hello-cert-trust-policy-settings.md`).

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by e0i** [comment]
@MK-Kamal 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Comment by e0i** [comment]
@Dansimp Would you merge the PR that has been approved by Matthew Palko? Thanks.

**Comment by DaniHalfin** [comment]
@e0i - this PR is currently in conflict. If the conflict is resolved, we'd be able to merge.

**Comment by illfated** [comment]
@MK-Kamal : Are you able to find the "Resolve conflict" editing tools to amend the changes to this file and get it updated to current version?

**Comment by MK-Kamal** [comment]
Sorry i delete the Path-6 by mistake

**Comment by MK-Kamal** [comment]
@illfated   

resolve Conflict is not activated 

image

**Comment by illfated** [comment]
Fair enough, thank you for doing your best with what you had available to you.
Your PR got merged, so the diff must have been manageable after all.

**Pull request merged**